cellprofiler/modules/editobjectsmanually.py

```diff
@@ -43,7 +45,9 @@
 
 See also **FilterObjects**, **MaskObject**, **OverlayOutlines**,
 **ConvertToImage**.
-"""
+""".format(**{
+    "HELP_ON_SAVING_OBJECTS": _help.HELP_ON_SAVING_OBJECTS
```

Comment:
I don't see any corresponding `{HELP_ON_SAVING_OBJECTS}` in the documentation string. Is it missing (e.g., should there be a block of text on saving objects in this module's documentation)? If not, this `.format({...})` is unnecessary.